### DISCUSSION TITLE
upgrding 1.21.0 to 1.23.0 facing issus , lower env worked fine but prodction level we facing issue .

### DISCUSSION THREAD

**Discussion by vrk7757** [istio/istio#53941] [Q&A]
hi team , iam upgrading istio version from 1.21.0 to 1.23.0 we are facing attached  issue on installation. please help . iam attaching installation-steps and installation-error, pod logs and scenario
[istio-error.txt](https://github.com/user-attachments/files/17814177/istio-error.txt)
[istio-installation-steps.txt](https://github.com/user-attachments/files/17814178/istio-installation-steps.txt)

**Comment by zirain** [comment]
please don't resubmit same issue with https://github.com/istio/istio/issues/53921
